scripts/source-maintenance.sh

```diff
@@ -320,65 +319,31 @@ echo "#endif"
 ) >lib/profiler/list
 mv lib/profiler/list lib/profiler/xprof_type.h
 
+amFileGenerate ()
```

Comment:
I suggest renaming this function to avoid Yoda words order and emphasize that the function does not create any files:

```suggestion
printAmFile ()
```

or 

```suggestion
printAutomakefile ()
```

I do not insist on this change.

Reply:
Done in 2516080